test/test_app.cpp

```diff
@@ -6,9 +6,9 @@
  *  SPDX-License-Identifier: LGPL-3.0-or-later
  ****************************************************************************/
 
-
 #include "app/app.h"
 
+#include <array>
```

Comment:
Interesting, do we really need this?

Reply:
Yes, otherwise this does not compile anymore with GCC 12. It is used here:
https://github.com/morxa/tacos/blob/db56d9085a0bfb13be1ccdc482fd00f7d3b3ba65/test/test_app.cpp#L25

Before, this was implicitly included by some other include, but that was apparently restructured in GCC.